executor/executor.cc

```diff
@@ -95,6 +95,7 @@ const int kOutFd = 4;
 static uint32* output_data;
 static uint32* output_pos;
 static uint32* write_output(uint32 v);
+static uint32* write_output_64(uint64 v);
```

Comment:
I'm not sure about the name. Maybe we should use function overloading here but i decided against because 
it easy to trip up and pass uint32 instead of uint64 and compiler will not warn about it. What do you think ?

Reply:
Let's go with this version for now. I don't see significant reasons to switch to something else now.

Reply:
done